Configuration/Settings.yaml

```diff
@@ -7,6 +7,7 @@ TYPO3:
           FacebookOAuth2Provider:
             provider: Flowpack\OAuth2\Client\Provider\FacebookProvider
             providerOptions:
+              #authenticateRoles: ['TYPO3.Flow.SomeRole']
               authorizationEndpointUri: 'https://www.facebook.com/dialog/oauth'
               redirectionEndpointUri: 'https://www.wishbase.com/authentication/authenticate'
```

Comment:
haha; since you're on it @mgoldbeck could you nuke the ``wishbase`` occurrences in this repo please? :)